pkgs/development/python-modules/ez_setup/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+}:
+buildPythonPackage rec {
```

Comment:
Not sure whether you need `rec` here.

Reply:
Thanks for the feedback. I removed `rec`.

Reply:
Hey for whatever reason the package breaks when `rec` is removed. I suspect that it has to do with the use of `inherit`?